rules/linux/execution_cpulimit_binary.toml

```diff
@@ -0,0 +1,52 @@
+[metadata]
+creation_date = "2022/03/17"
+maturity = "production"
+updated_date = "2022/03/17"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies Linux binary cpulimit abuse to break out from restricted environments by spawning an interactive system
+shell.The cpulimit utility is used to restrict the CPU usage of a process in cases of CPU or system load exceeding the
+defined threshold and the activity of spawing a shell is not a standard use of this binary by a user or system
+administrator. It indicates a potentially malicious actor attempting to improve the capabilities or stability of their
+access.
```

Comment:
```suggestion
Identifies Linux binary cpulimit abuse to break out from restricted environments by spawning an interactive system
shell. The cpulimit utility is used to restrict the CPU usage of a process in cases of CPU or system load exceeding the
defined threshold and the activity of spawning a shell is not a standard use of this binary by a user or system
administrator. This can potentially indicate a malicious actor attempting to improve the capabilities or stability of their
access.
```